AppServices/Orders/OrderAppService.cs

```diff
@@ -0,0 +1,67 @@
+﻿using AppModels.Orders.Mapper;
+using AppServices.Orders.Interface;
+using AppServices.Products.Interface;
+using AutoMapper;
+using DomainModels;
+using DomainServices.Orders.Interface;
+
+namespace AppServices.Orders
+{
+    public class OrderAppService : IOrderAppService
+    {
+        private readonly IOrderService _orderService;
+        private readonly IMapper _mapper;
+        private readonly IProductAppService _productAppService;
+
+        public OrderAppService(IOrderService orderService, IMapper mapper, IProductAppService productAppService)
+        {
+            _orderService = orderService ?? throw new ArgumentNullException(nameof(orderService));
+            _mapper = mapper ?? throw new ArgumentNullException(nameof(mapper));
+            _productAppService = productAppService ?? throw new ArgumentNullException(nameof(productAppService));
+        }
+        public IEnumerable<OrderResult> GetByPortfolioId(long portfolioId)
+        {
+            var orderList = _orderService.GetByPortfolioId(portfolioId);
+
+            return _mapper.Map<IEnumerable<OrderResult>>(orderList);
+        }
+
+        public async Task<long> CreateAsync(CreateOrderRequest createOrder)
+        {
+            var checkProduct = await _productAppService.GetByIdAsync(createOrder.ProductId);
+            var mapOrder = _mapper.Map<Order>(createOrder);
+
+            if (checkProduct == null)
+            {
+                throw new ArgumentException($"Product not found with this ProductId. ProductId: {createOrder.ProductId}");
+            }
+            mapOrder.UnitPrice = checkProduct.UnitPrice;
+            mapOrder.NetValue = mapOrder.Quotes * mapOrder.UnitPrice;
+
+            return await _orderService.CreateAsync(mapOrder).ConfigureAwait(false);
+        }
+        public async Task<OrderResult> GetByIdAsync(long id)
+        {
+            var order = await _orderService.GetByIdAsync(id).ConfigureAwait(false);
+
+            return _mapper.Map<OrderResult>(order);
+        }
+
+        public int NumberOfAvaiableQuotes(long portfolioId, long productId)
+        {
+            return _orderService.NumberOfAvaiableQuotes(portfolioId, productId);       
+        }
+
+        public void Delete(long id)
+        {
+            _orderService.Delete(id);
+        }
+
+        public IEnumerable<OrderResult> GetExecutableOrders()
+        {
+            var orders = _orderService.GetExecutableOrders();
```

Comment:
Não estamos retornando apenas 1
```suggestion
        public IEnumerable<OrderResult> GetAllExecutableOrders()
        {
            var orders = _orderService.GetAllExecutableOrders();
```